ext/bcmath/libbcmath/src/str2num.c

```diff
@@ -38,9 +38,8 @@
 bool bc_str2num(bc_num *num, char *str, size_t scale)
 {
 	size_t digits = 0;
-	size_t strscale = 0;
-	char *ptr, *nptr;
-	size_t trailing_zeros = 0;
+	size_t str_scale = 0;
+	char *ptr, *nptr, *integer_ptr, *integer_end, *fractional_ptr = NULL, *fractional_end = NULL, *decimal_point;
```

Comment:
I usually prefer to have each declaration on its own line

Reply:
In release builds this had no effect so I reverted it